package.json

```diff
@@ -22,8 +22,7 @@
   "homepage": "https://github.com/hashicorp/js-releases#readme",
   "dependencies": {
     "@types/semver": "^7.3.1",
-    "del": "^5.1.0",
-    "openpgp": "5.0.0-1",
+    "openpgp": "5.0.0",
```

Comment:
Looks like `5.0.1` was published just yesterday - is it worth bumping to that one instead?